src/main/java/com/google/codeu/data/User.java

```diff
@@ -0,0 +1,20 @@
+package com.google.codeu.data;
+
+public class User {
```

Comment:
You probably should add a javadoc comment to indicate what you think the usage of this class will be.